sdk/azidentity/managed_identity_credential.go

```diff
@@ -75,7 +75,7 @@ func (c *ManagedIdentityCredential) GetToken(ctx context.Context, opts azcore.To
 }
 
 // AuthenticationPolicy implements the azcore.Credential interface on ManagedIdentityCredential.
-// Please note: the TokenRequestOptions included in AuthenticationPolicyOptions must be a slice of resources in this case and not scopes
+// Please note: the TokenRequestOptions included in AuthenticationPolicyOptions must be a slice of resources in this case and not scopes.
```

Comment:
I feel like `NOTE:` stands out a bit more and is what I've done in other docs.

Reply:
Done